internal/consensus/pbts_test.go

```diff
@@ -454,10 +471,10 @@ func TestPBTSTimelyProposal(t *testing.T) {
 			Precision:    10 * time.Millisecond,
 			MessageDelay: 140 * time.Millisecond,
 		},
-		timeoutPropose:             40 * time.Millisecond,
-		genesisTime:                initialTime,
-		height2ProposedBlockTime:   initialTime.Add(10 * time.Millisecond),
-		height2ProposalDeliverTime: initialTime.Add(30 * time.Millisecond),
+		timeoutPropose:                    40 * time.Millisecond,
+		genesisTime:                       initialTime,
+		height2ProposedBlockOffset:        15 * time.Millisecond,
```

Comment:
We are replacing 10ms by 15ms. Should not change much, I assume, but...

Reply:
the more we can reduce time-driven tests the better....

Reply:
We are mocking the clock, so this should be that bad.

Reply:
Didn't realize, thank you for explaining 🙂